src/app/validation/commandValidation.ts

```diff
@@ -101,19 +101,25 @@ const paid = async (request: PaidRequest): Promise<void> => {
     const dbCollectionBounties = db.collection('bounties');
     const dbBountyResult: BountyCollection = await dbCollectionBounties.findOne({
         _id: new mongo.ObjectId(request.bountyId),
-        isIOU: true,
     });
 
     if (!dbBountyResult) {
         throw new ValidationError(
-            `Please select a valid IOU id to mark paid. `
+            `Please select a valid bounty to mark paid. `
         );
     }
 
-    if (dbBountyResult.status && dbBountyResult.status !== BountyStatus.open) {
+    const iouStatus = dbBountyResult.status && 
+        dbBountyResult.status !== BountyStatus.open && 
+        dbBountyResult.isIOU;
+    const bountyStatus = dbBountyResult.status && 
```

Comment:
Maybe name this bountyNotPayable instead of bountyStatus. bountyStatus sounds like it just holds the current status. Same above: iouStatus -> iouNotPayable